sdk/healthinsights/azure-health-insights-radiologyinsights/src/main/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsClient.java

```diff
@@ -272,160 +282,313 @@ public final class RadiologyInsightsClient {
+     *         ]
+     *         innererror (Optional): {
+     *             code: String (Optional)
+     *             innererror (Optional): (recursive schema, see innererror above)
+     *         }
+     *     }
      * }
      * }</pre>
      *
-     * @param radiologyInsightsData Contains the list of patients, and configuration data.
+     * @param id The unique ID of the job.
+     * @param resource The resource instance.
      * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
      * @throws HttpResponseException thrown if the request is rejected by server.
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
-     * @return the {@link SyncPoller} for polling of provides status details for long running operations.
+     * @return the {@link SyncPoller} for polling of response for the Radiology Insights request.
      */
     @Generated
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public SyncPoller<BinaryData, BinaryData> beginInferRadiologyInsights(BinaryData radiologyInsightsData,
+    public SyncPoller<BinaryData, BinaryData> beginInferRadiologyInsights(String id, BinaryData resource,
         RequestOptions requestOptions) {
-        return this.serviceClient.beginInferRadiologyInsights(radiologyInsightsData, requestOptions);
+        return this.serviceClient.beginInferRadiologyInsights(id, resource, requestOptions);
+    }
+
+    /**
+     * Create Radiology Insights job
+     *
+     * Creates a Radiology Insights job with the given request body.
+     *
+     * @param id The unique ID of the job.
+     * @param resource The resource instance.
+     * @param expand Expand the indicated resources into the response.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the {@link SyncPoller} for polling of response for the Radiology Insights request.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    public SyncPoller<RadiologyInsightsJob, RadiologyInsightsInferenceResult> beginInferRadiologyInsights(String id,
+        RadiologyInsightsJob resource, List<String> expand) {
```

Comment:
What's the decision about Srikanta's comment here?
https://apiview.dev/Assemblies/Conversation/c48b47bd12a6493aa27b2cec58f9e54e#46f262e257de451485c82b43abb61bb5
(this can be easily customized)

However, service apperas having a new `expand` on the API. You may have to make an option class.